### ISSUE TITLE
Download on firefox forces a popup

### ISSUE THREAD

**Issue by arogl** [zluo01/aria2-extension#978]
Is there any to match the behaviour of this chrome extension of just taking over the downloads without the need of a popup?

[Aria2 Explorer](https://github.com/alexhua/Aria2-Explorer)

Or is there an option I have not set?

Any help appreciated, finding a replacement downloader like the above is all that is really stopping me swapping to Firefox permanently.

**Comment by zluo01** [comment]
No, there is no such option. I do not have time until mid Dec to add this feature if necessary. At the mean time, change should be very easy (add a new config options and bypass create pop up here https://github.com/zluo01/aria2-extension/blob/main/src/background/index.ts#L103.) You can create a PR with this change.
